style.css

```diff
@@ -0,0 +1,60 @@
+*
+    {
+        font-family: Arial, Helvetica, sans-serif;
+        color: white;
+        background-color: #181818;
+    }
+    
+    nav ul 
+    {
+        display: flex;
+        list-style-type: none;
+        justify-content: center;
+    }
+
+    nav ul li
+    {
+        margin: 0 20px;
+    }
+
```

Comment:
I think it would be a nice feature to remove the underline on the navbar links using `text-decoration: none`, but you could show them again on `:hover` with `text-decoration: underline`.